pkg/controllers/provisioningv2/cluster/controller.go

```diff
@@ -45,6 +45,7 @@ const (
 	administratedAnn      = "provisioning.cattle.io/administrated"
 	mgmtClusterNameAnn    = "provisioning.cattle.io/management-cluster-name"
 	fleetWorkspaceNameAnn = "provisioning.cattle.io/fleet-workspace-name"
+	turtlesOwnedLab       = "cluster-api.cattle.io/owned"
```

Comment:
are you using `cluster-api.cattle.io` in other places? can we actually use `provisioning.cattle.io` as the label domain for this so that we maintain consistency? 

you could call it `provisioning.cattle.io/externally-managed`